R/fullEXAMPLE.R

```diff
@@ -33,5 +33,6 @@ fullEXAMPLE <- function(rev = 0, dev = "", extra = "Example argument") {
     message("Here you could execute code for a hypothetical development version called \"test\"")
   }
   # return is not required, but can be used to append a tag to the resulting filename
-  return(list(tag = "customizable_tag"))
+  return(list(tag = "customizable_tag",
+              pucTag = "tag"))
```

Comment:
What is the pucTag doing?

Reply:
same as `tag` but for put files...I add some information about it to the documentation of `retrieveData`